setup.py

```diff
@@ -1,14 +1,15 @@
 import pathlib
 
-from setuptools import setup
+from setuptools import find_packages, setup
+
+packages = find_packages(exclude=["examples", "examples.*", "tests", "tests.*"])
 
-# TODO: Add auto discovery for packages.
 setup(
     name="lbz",
     version=pathlib.Path("version").read_text("utf-8").strip(),
     author="Piotr Dyba",
     author_email="piotr.dyba@localbini.com",
-    packages=["lbz", "lbz.authz", "lbz.dev", "lbz.events", "lbz.lambdas"],
+    packages=packages,
```

Comment:
I don't think we need to create a separate variable for that, at least we don't do so for other values:
```suggestion
    packages=find_packages(exclude=["examples", "examples.*", "tests", "tests.*"]),
```